### DISCUSSION TITLE
Give a YT link to comments

### DISCUSSION THREAD

**Discussion by Papoulka** [mattwright324/youtube-comment-suite#118] [Ideas]
This is a great tool of course.

As an improvement:  provide the actual YT link to comments.  I can find comments in a Search, and can "View Thread"... but that pops a local viewer.  What I often want to do is find the comment on YT itself.  Other than a date there is no info on that - not even to the relevant video.  Nor in an "Export" of the results.

Thank you

**Comment by mattwright324** [comment]
Hi @Papoulka 

This is actually a thing already! If you right click on a comment in the list there is an option to `Open in Browser` which opens a direct link of the comment in your default browser the way you are describing. There is also `Copy Comment Link(s)` which puts the link(s) in your clipboard instead of just opening in the browser.

![image](https://user-images.githubusercontent.com/7717432/183310579-fa966086-f257-4d92-87e5-4a3c1fc5340f.png)

**Comment by Papoulka** [comment]
My mistake!  Something in my brain "knew" that there was a way to do it, but I couldn't find it later.  Thanks for the quick response.

**Comment by Papoulka** [comment]
Ah - but can you include the link in the Export?  That would be very helpful for getting back to a comment later.  And much more useful than the link to the poster's thumbnail.

Thank you.
